app/src/main/res/layout/dialog_layout.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+   >
```

Comment:
Please move the closing tag to the previous line. Most other elements in this file already do it with the consistent style. Also, please use one single attribute per line.